src/scss/includes/panels/service_panel.scss

```diff
@@ -52,6 +51,18 @@
     h3 {
       margin-bottom: 6px;
     }
+
+    .panel-header {
+      padding: 7px 0 0;
```

Comment:
Shouldn't we use 4, 8, 16, 24 px only ?

Reply:
I didn't know we had this kind of rule, but sure, 8px works too